test/Typewriter.Test/TestDataPHP/com/Microsoft/Graph/CallRecords/Model/CallRecord.php

```diff
@@ -206,15 +206,15 @@ public function setEndDateTime($val)
     /**
     * Gets the organizer
     *
-    * @return Microsoft\Graph\Model\IdentitySet The organizer
+    * @return \Microsoft\Graph\Model\IdentitySet The organizer
     */
     public function getOrganizer()
     {
         if (array_key_exists("organizer", $this->_propDict)) {
```

Comment:
Should we also take this opportunity to factor in some changes from a community member ?(https://github.com/microsoftgraph/msgraph-sdk-php/pull/156) considering it affects many files

We can add the `$this->getProperties()` to the template

Reply:
Hey @Ndiritu, thanks for the heads-up, will be making the necessary changes and do more improvements to the template file.

Reply:
I would recommend taking these fixes separately so that the commit history will be easy to understand for anyone looking back. It is even more useful when one of the fixes causes a bug as a side effect and if it needs to be reverted because of that.

I would recommend batching for breaking changes to release in a single major version, but even then having separate PRs helps a lot in the scenario described above.

Reply:
Yes, that makes sense @zengin